### PR TITLE
feat(66937177): Add Homepage and Controller

### PR DESCRIPTION
**Pull request by Maxiboy441**

### CODE REVIEW

**Comment by Maxiboy441** [comment]
Info: Draft, forgot smth

**Comment by Maxiboy441** [comment]
Can be reviewed

File: Project/Project/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -1,8 +1,99 @@
...
+        width: 100px;
+        font-weight: bold;
+    }
 
+    .carousel-caption {
+        background: rgba(0, 0, 0, 0.7);
+        padding: 20px;
+        border-radius: 10px;
+    }
+</style>
```
Comment (feliperamoss) [inline Project/Project/Views/Home/Index.cshtml:99]: Please extract styles to css file

File: Project/Project/Controllers/HomeController.cs [hunk 2]
```diff
@@ -1,20 +1,61 @@
...
+                aggregatedRatings.Add(averageRating);
+            }
+        }
+
+        var topRated = aggregatedRatings
+            .OrderByDescending(r => r.Rating_value)
+            .Take(10)
+            .ToList();
+
+        return topRated;
```
Comment (feliperamoss) [inline Project/Project/Controllers/HomeController.cs:53]: Can you try if the code below works? As it shorter and simpler:
public List<Rating> TopRated()
{
    // Aggregating ratings and calculating the average rating for each movie
    var topRatedMovies = _context.Ratings
        .GroupBy(r => new { r.Movie_id, r.Movie_title, r.Movie_poster })
        .Select(g => new Rating
        {
            Movie_id = g.Key.Movie_id,
            Movie_title = g.Key.Movie_title,
            Movie_poster = g.Key.Movie_poster,
            Rating_value = Math.Round(g.Average(r => r.Rating_value), 1),
            Created_at = DateTime.Now,
            Updated_at = DateTime.Now
        })
        .OrderByDescending(r => r.Rating_value)
        .Take(10)
        .ToList();

    return topRatedMovies;
}

**Pull request merged**